.github/tests.yml

```diff
@@ -0,0 +1,30 @@
+name: Tests
+
+on: [push]
+
+jobs:
+  unit:
+    name: unit
+    runs-on: ubuntu-16.04
+    steps:
+      - uses: actions/checkout@v1
+      with:
+        python-version: 3.7
```

Comment:
Do you want to allow for tests in 3.6 and 3.7?

Reply:
Hmm actually I think this one is replaced by the workflow below. I'll try removing, which will also stabilize ubuntu versions.